packages/twenty-front/src/testing/mock-data/peopleV2.ts

```diff
@@ -0,0 +1,533 @@
+import { Company } from '@/companies/types/Company';
+import { Person } from '@/people/types/Person';
+
+export type MockedPersonV2 = Pick<
```

Comment:
why do we have a V2? Should we remove the other one?
We should not introduce V2 if there is an "easy way" to clean the v1. V2 is only relevant for big refactoring cases IMO

Reply:
Refactored in next PR